docs/source/data/subscriptions.mdx

```diff
@@ -366,159 +393,154 @@ subscription IncrementingNumber {
 }
 ```
 
-After you start up the server in CodeSandbox, follow the instructions in the browser to test out running the `numberIncremented` subscription in Apollo Sandbox. You'll probably need to configure the subscriptions endpoint (`wss://YOUR_SANDBOX_ID.sse.codesandbox.io/graphql`) in Sandbox's Settings tab. You'll see the subscription's value update every second.
+After you start up the server in CodeSandbox, follow the instructions in the browser to test running the `numberIncremented` subscription in Apollo Sandbox. You should see the subscription's value update every second.
+
+> If you don't see the result of your subscription operation you might need to [adjust your Sandbox settings to use the `graphql-ws` protocol](https://www.apollographql.com/docs/studio/explorer/additional-features/#subscription-support).
 
 ## Operation context
 
 When [initializing context](../data/resolvers/#the-context-argument) for a query or mutation, you usually extract HTTP headers and other request metadata from the `req` object provided to the `context` function.
 
-**For subscriptions,** you provide an `onConnect` function to the `SubscriptionServer` constructor. This function receives the [`connectionParams`](#connectionparams) in an object as the first argument, as well as the [`WebSocket`](#websocket) and [`ConnectionContext`](#context) as the second and third arguments, respectively. If your `onConnect` function returns an object, it's passed to your resolvers as the `context` argument.
-
-```js
-SubscriptionServer.create({
-  // ...other options
-  async onConnect(connectionParams) {
-    if (connectionParams.authorization) {
-      const currentUser = await findUser(connectionParams.authorization);
-      return { currentUser };
-    }
-    throw new Error('Missing auth token!');
-  }
-});
-```
-
-This is especially important because metadata like auth tokens are sent differently depending on the transport.
-
-## `onConnect` and `onDisconnect`
+**For subscriptions,** you can extract information from a client's request by adding [options](https://github.com/enisdenjo/graphql-ws/blob/master/docs/interfaces/server.ServerOptions.md) to the first argument passed to the `useServer` function.
 
-You can define functions that the subscription server executes whenever a subscription request connects (`onConnect`) or disconnects (`onDisconnect`).
+For example, you can supply a [`context` property](https://github.com/enisdenjo/graphql-ws/blob/master/docs/interfaces/server.ServerOptions.md#context) to add values of your GraphQL operation [`context`](./resolvers/#the-context-argument):
```

Comment:
supply -> provide

"add values of" -> "add values to", yeah?

Reply:
Oooh yes, good catch!